src/lib/extensions/SapphireMessage.ts

```diff
@@ -0,0 +1,34 @@
+import type { Awaited } from '@sapphire/pieces';
+import { Message, MessageAdditions, MessageOptions, SplitOptions, Structures } from 'discord.js';
+
+export class SapphireMessage extends Structures.get('Message') {
```

Comment:
Any reason for this over `Structures#extend`?

Reply:
To export `SapphireMessage`.